src/library/autodj/autodjprocessor.cpp

```diff
@@ -155,9 +155,8 @@ AutoDJProcessor::AutoDJProcessor(
 
     // TODO(rryan) listen to signals from PlayerManager and add/remove as decks
```

Comment:
I know it's not related to your change, but anything we can do about this 10yo+ TODO?